cg/server/api.py

```diff
@@ -398,8 +399,11 @@ def get_sequencing_metrics(flow_cell_name: str):
             jsonify({"error": f"Sequencing metrics not found for flow cell {flow_cell_name}."}),
             HTTPStatus.NOT_FOUND,
         )
-
-    return jsonify([metric.to_dict() for metric in sequencing_metrics])
+    metrics_dtos: list[SequencingMetricsRequest] = [
+        SequencingMetricsRequest.model_validate(metric, from_attributes=True)
+        for metric in sequencing_metrics
+    ]
+    return jsonify([metric.to_dict() for metric in metrics_dtos])
```

Comment:
I think the reason why we used `.dict` was because it was an overridden method in the db models. I think for a DTO it would be better to use `model_dump`